en/simulation/ignition_gazebo.md

```diff
@@ -0,0 +1,50 @@
+# Ignition Gazebo Simulation
+
+[Ignition Gazebo](https://ignitionrobotics.org/libs/gazebo) is an open source robotics simulator from the [Ignition Robotics Project](https://ignitionrobotics.org/home).
+Ignition Gazebo is derived from the popular robotics simulator [Gazebo](./gazebo.md), featuring more advanced rendering, physics and sensor models.
+
+**Supported Vehicles:** Quadrotor
+
+@[youtube](https://youtu.be/38UJqrNQChg)
+
+:::note
+See [Simulation](../simulation/README.md) for general information about simulators, the simulation environment, and simulation configuration (e.g. supported vehicles).
+:::
+
+## Installation (Ubuntu Linux)
+
+:::note
+These instructions were tested on Ubuntu 18.04 and Ubuntu 20.04
+:::
+
+1. Install the usual [Development Environment on Ubuntu LTS / Debian Linux](../dev_setup/dev_env_linux_ubuntu.md).
+1. Install Ignition Gazebo following the installation [instructions]()
+   ```sh
+   sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
```

Comment:
1. For me all of this needed prefix of `sudo`. Should we add that?
2. Will we be testing ignition gazebo regularly? If so, can we get this added to the setup script?